src/components/CounterBtn.jsx

```diff
@@ -0,0 +1,12 @@
+function CounterBtn({cls, bodyText, onClick, disable}){
+
+    return(
+        <>
+            <button className={cls} onClick={onClick} disabled={disable}>
+                {bodyText}
+            </button>
+        </>
+    )
+}
+
```

Comment:
Dit component is nu een soort van doorgeefluik. Je had net zo goed direct het button element kunnen gebruiken, omdat het je geen code bespaart

Reply:
Dank je wel voor de feedback Zwen, Ik geloof dat dit een van de opdrachten was. Daarnaast was het voor mij een goede oefening om components te bouwen. Maar ik snap je punt uiteraard